debug-bar.php

```diff
@@ -125,6 +125,10 @@ function enqueue() {
 
 		wp_enqueue_script( 'debug-bar', plugins_url( "js/debug-bar$suffix.js", __FILE__ ), array( 'jquery' ), '20170623', true );
 
+		if ( defined( 'SAVEQUERIES' ) && SAVEQUERIES || ! empty( $GLOBALS['EZSQL_ERROR'] ) ) {
```

Comment:
Is there a particular reason you choose not to implement this with a helper method ? I previously suggested this in https://github.com/dd32/debug-bar/pull/40#discussion_r132350474 ?

Reply:
Wasn't sure how to implement, then forgot to go back.
I have put a public helper function in the Debug_Bar class.
It seems to work well enough. Not sure if the way I called the helper in the prerender function of the queries panel is the appropriate way to do it.